var/spack/repos/builtin/packages/py-pacifica-downloader/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPacificaDownloader(PythonPackage):
+    """Python Pacifica Download Library"""
+
+    homepage = "https://github.com/pacifica/pacifica-python-downloader/"
+    pypi     = "pacifica-downloader/pacifica-downloader-0.4.1.tar.gz"
+
+    version('0.4.1', sha256='11da2032a07ca7bb06fed38dc8d7c4c57267ff98c5fd925271083e18dd85d9f4')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
+    depends_on('py-requests', type=('build', 'run'))
+    depends_on('py-pacifica-namespace', type=('build', 'run'))
+    depends_on('python@3:', type=('build', 'run'))
```

Comment:
Where do you see this requirement?

Reply:
mostly because they dropped all testing for python2

Reply:
I would suggest dropping this because the supported Python versions aren't explicitly enforced in the setup.py.

Reply:
Also missing build dep on setuptools-scm